website/src/MDPage.js

```diff
@@ -29,9 +27,20 @@ const getHeadingForLevel = (level) => {
   }
 };
 
+const isDefined = (obj) => obj !== undefined;
+
+const getDocsMD = (docPath, navigation) => {
+  const { state } = navigation;
+  const docVersion = (isDefined(state) && isDefined(state.params))
+    ? state.params.version
+    : 'edge';
```

Comment:
That logic is already defined in two places - here and in version selector. We could either use `initialRouteParams` for cases when params are not set?